script.100fm/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category>
+        <setting label="10000" type="number" id="channel" default="0" visible="false"/>
```

Comment:
Sure, again - it caused because of no default value was set.
What's wrong with having this file anyway? it helps me to track which variables are persistent.